line_profiler/_line_profiler.pyx

```diff
@@ -33,13 +71,22 @@ cdef extern from "Python.h":
     cdef int PyTrace_C_EXCEPTION
     cdef int PyTrace_C_RETURN
 
-cdef extern from "timers.h":
+cdef extern from "timers.c":
     PY_LONG_LONG hpTimer()
     double hpTimerUnit()
 
-cdef extern from "unset_trace.h":
+cdef extern from "unset_trace.c":
     void unset_trace()
 
+cdef struct LineTime:
```

Comment:
I put this into a struct so that I could put it into the unordered_map. I don't believe a cdef class can go into an unordered_map, though it may be worth a shot.